src/ex2/View.js

```diff
@@ -0,0 +1,196 @@
+export default class View {
+  constructor(addItemHandler, deleteItemHandler, getAllItemsHandler) {
+    this.toDoInput = document.querySelector("input");
+    this.listContainer = document.querySelector("ul");
+    this.alertBox = document.querySelector(".alert");
+    this.alertBoxText = document.querySelector(".alert-innet-text");
+    this.addItemHandler = addItemHandler;
```

Comment:
instead of passing props you could have pass an entire instance of `ItemManager`